internal/services/patch_anchor_platform_transactions_completion.go

```diff
@@ -7,14 +7,15 @@ import (
 	"github.com/stellar/go/support/log"
 	"github.com/stellar/stellar-disbursement-platform-backend/internal/anchorplatform"
 	"github.com/stellar/stellar-disbursement-platform-backend/internal/data"
+	"github.com/stellar/stellar-disbursement-platform-backend/internal/db"
 )
 
-type PatchAnchorPlatformTransactionService struct {
+type PatchAnchorPlatformTransactionServiceCompletion struct {
```

Comment:
This name is incorrect. It's about the TransactionCompletion and not the ServiceCompletion, right?

Reply:
My bad! You're right. Fixing it.